src/renderer/components/ft-prompt/ft-prompt.js

```diff
@@ -36,7 +36,11 @@ export default defineComponent({
     autosize: {
       type: Boolean,
       default: false
-    }
+    },
+    isFirstOptionDestructive: {
```

Comment:
I think it's better to have `destructiveOptionValues`
1. It might not be only 1st option being destructive
2. Destructive options might be ordered later (normal action, close, destructive action)

Reply:
That's interesting, but seems premature until we have a matching use case - it seems reasonable to keep destructive action first as a rule for the sake of consistency & user expectation.